packages/core/core/src/public/Bundle.js

```diff
@@ -168,20 +184,38 @@ export class NamedBundle extends Bundle implements INamedBundle {
   #bundleGraph; // BundleGraph
 
   constructor(
+    sentinel: mixed,
```

Comment:
This prevents anything but something in this file from invoking this constructor 😄